intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/requirements.txt

```diff
@@ -14,4 +14,6 @@ qdrant-client
 rank_bm25
 scikit-learn
 sentence-transformers==2.3.1
+torch==2.2.0
```

Comment:
is 2.2 a must? can be 2.1 or less?

Reply:
just align the ITREX requirements for torch. Otherwise some dependency in this requirements.txt may downgrade the torch to 2.1